src/pages/AccountPage.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { Alert, StyleSheet, Text, ScrollView, ImageBackground } from 'react-native';
+import { AccountInput } from "../components/input"
+import { UpdateAccountButton } from "../components/button"
+import { UserStore } from '../stores/UserStore';
+import { styleConstants } from '../config/constants';
+import { requiredFieldsEmpty } from '../utilities/FormValidation';
+
+interface Props {
+    userStore: UserStore;
+}
+
+interface State {
+    email?: string;
+    password?: string;
+    firstName?: string;
+    lastName?: string;
+    phoneNumber?: string;
+}
+export class AccountPage extends React.Component<Props, State> {
+    constructor(props: Props) {
+        super(props);
+
+        this.state = {
+            email: '',
+            password: '',
+            firstName: '',
+            lastName: '',
+            phoneNumber: '',
+        };
+    }
+
+    private onPressSignUpButton = (): void => {
+        if (false === this.validateInputs()) {
+            return;
+        }
+
+        const { email, password, firstName, lastName, phoneNumber } = this.state;
+    };
+
+    private validateInputs(): boolean {
```

Comment:
Can we use the method u created for this?

Reply:
This one alerts the user specifically what's missing instead of just doing the logic.  I could come up with a generic alert that just tells them the * fields are required if you like.